tests/cli-tests/compression/window-resize.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+datagen -g1G > file
+zstd --long=31 -1 --single-thread --no-content-size -f file
+zstd -l -v file.zst
+
+# We want to ignore stderr (its outputting "*** zstd command line interface
```

Comment:
One way to silence `stderr` could be `2> /dev/null` (redirects `stderr` to `/dev/null`)